src/docs/predicted_molecular_consequence.md

```diff
@@ -14,7 +14,7 @@ A set of information enabling interpretation of likely variant impact on genomic
 | variant_representation | array of VariantRepresentation or [ ]  | Nomenclature values and source
 | prediction_results | array of PredictionResult or [ ] | Scores from programs like SIFT which calculate transcript-specific deleteriousness scores
 | transcript_biotype | string | Feature biotype
-| cdna_location | VariantRelativeLocation  | Relative location on cDNA
+| cdna_location | VariantRelativeLocation or null  | Relative location on cDNA
```

Comment:
I just want to point out that at the moment, there is a contradiction between the `feature_type` field, which allows values other than transcript, and the rest of the fields that _only_ make sense for transcripts (transcript_biotype, cds_location, cdna_location, etc.)

Reply:
Addition to that regulatory feature mention in the description, `intergenic` variant types will also not have value for those fields.

But both regulatory and intergenic type variants are not part of MVP. We need to think about modelling them in the post-MVP era.

Reply:
If we plan to model something like`transcript_predicted_molecular_consequences` , `regulatory_predicted_molecular consequences` and the client knows that transcript_predicted_molecular_consequences always relate to the feature "Transcript" and its corresponding SO term without the use of `feature_type`, then it is redundant.

Reply:
you could have an enum of consequences (`VariantTranscriptConsequence`, `VariantRegulatoryConsequence`, etc.) in the same array of predicted molecular consequences; but in that case, the shape of `VariantRegulatoryConsequence` would be different from the shape of `VariantTranscriptConsequence`. E.g. the `transcript_biotype` won't make sense in `VariantRegulatoryConsequence`.

Reply:
Thanks @likhitha-surapaneni @azangru , 
I think we should have a separate discussion on accommodating regulatory consequence into the VDM post-mvp.
I will merge this PR as it satisfies the MVP case.

Reply:
Thanks @nakib103 , created a ticket for the same: [ENSVAR-6250](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6250)